zaza/openstack/charm_tests/keystone/tests.py

```diff
@@ -404,6 +404,14 @@ def _get_ldap_config(self):
             'ldap-password': 'crapper',
             'ldap-suffix': 'dc=test,dc=com',
             'domain-name': 'userdomain',
+            'ldap-user-tree-dn': 'ou=users,dc=test,dc=com',
+            'ldap-user-filter': '',
+            'ldap-user-objectclass': 'inetOrgPerson',
+            'ldap-user-id-attribute': 'cn',
+            'ldap-user-name-attribute': 'sn',
+            'ldap-group-tree-dn': 'ou=UserGroups',
+            'ldap-group-objectclass': 'groupOfNames',
+            'ldap-group-id-attribute': 'cn',
```

Comment:
Will this break on previous revision charms if this test is run?  If so, then this will need to be used in a new test class and only called from the new version of the charm (in the tests.yaml).

Reply:
Yes, this change breaks the existing charm revisions. Thanks for the suggestion. I will update a new patch.

Reply:
Submitted a new patch with the changes